happi/audit.py

```diff
@@ -0,0 +1,424 @@
+"""
+This module defines the ``happi audit`` command line utility
+
+If the --file option is not provided, then it will take the path from
```

Comment:
I think this should be explained in the `--file` argument and is unnecessary up here.

Reply:
I agree - moved it in 1f87a91